src/mkdocs_api_autonav/plugin.py

```diff
@@ -104,7 +107,7 @@ def _module_markdown(self, parts: tuple[str, ...]) -> str:
         """
         mod_identifier = ".".join(parts)  # top_module.sub.sub_sub
         # create the actual markdown that will go into the virtual file
-        md = f"---\ntitle: {mod_identifier}\n---\n\n::: {mod_identifier}\n"
+        md = f"---\ntitle: {self._display_title(parts)}\n---\n\n::: {mod_identifier}\n"
```

Comment:
```suggestion
        md = f"---\ntitle: {self._display_title(parts)}\n---\n\n# {mod_identifier}\n\n::: {mod_identifier}\n"
```

Reply:
I just wanted to ask if there's been any progress with this? I'd be keen to try this feature in vLLM.

Reply:
just returned from a vacation (this PR was snuck into freetime :)) ... working on it today